src/Options.php

```diff
@@ -9,6 +9,11 @@ trait Options
     /** Get an associative array of [case name => case value]. */
     public static function options(): array
     {
-        return array_column(static::cases(), 'value', 'name');
+        $cases = static::cases();
+        if (reset($cases) instanceof \BackedEnum) {
```

Comment:
Why is the `reset()` needed? Does that return the first value?

I think PHP 8.1 added more functions for working with arrays, so we can use those instead of the older ones that also touch the array pointers.

Reply:
Yes, `reset()` returns the first item, but as it's an intexed array, we could probably just grab `[0]` - an enum with no cases is a problem all on its own, but in that case we'd just be returning an empty array no matter what.